powertools-batch/src/main/java/software/amazon/lambda/powertools/batch/builder/DdbBatchMessageHandlerBuilder.java

```diff
@@ -0,0 +1,36 @@
+package software.amazon.lambda.powertools.batch.builder;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
+import com.amazonaws.services.lambda.runtime.events.StreamsEventResponse;
+import software.amazon.lambda.powertools.batch.exception.DeserializationNotSupportedException;
+import software.amazon.lambda.powertools.batch.handler.BatchMessageHandler;
+import software.amazon.lambda.powertools.batch.handler.DdbBatchMessageHandler;
+
+import java.util.function.BiConsumer;
+
+public class DdbBatchMessageHandlerBuilder extends AbstractBatchMessageHandlerBuilder<DynamodbEvent.DynamodbStreamRecord,
```

Comment:
Shouldn't we call it `DynamoDbBatchMessageHandlerBuilder` ? A bit long but Ddb may not be clear...